lib/heroku_rails_deploy/deployer.rb

```diff
@@ -153,8 +150,11 @@ def run_heroku_command(app_name, command, validate: nil)
         # If we're running a shell command, return the underlying
         # shell command exit code
         cli_command << ' --exit-code'
+        display_output = true
```

Comment:
This is probably good enough for now but displaying output for shell commands vs. Heroku API commands is pretty subtle. Longer term we might want to have separate methods to make this more obvious.